Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -150,7 +163,9 @@ public function registerAssetChange(AssetInterface $asset)
                 continue;
             }
 
-            $this->registerChangeOnNodeIdentifier($reference->getNodeIdentifier());
+            $node = $this->getContextForReference($reference)->getNodeByIdentifier($reference->getNodeIdentifier());
+            $this->registerNodeChange($node);
+
             $this->registerChangeOnNodeType($reference->getNodeTypeName(), $reference->getNodeIdentifier());
```

Comment:
Can't we remove this line then as the `registerNodeChange` also triggers the `registerChangeOnNodeType` ?

Reply:
True, didn't see that, as it's two calls down the stack…